src/app/home/page.tsx

```diff
@@ -0,0 +1,52 @@
+'use client'
+import { HeroComponent } from '@/components/HeroComponent'
+import { useGetProposalsWithGraph } from '../proposals/hooks/useGetProposalsWithGraph'
+import { LatestCollectiveSection } from '../user/latest-collective'
+import { Paragraph } from '@/components/TypographyNew'
+import { ExternalLink } from '@/components/Link/ExternalLink'
+import { TreasuryContextProviderWithPrices } from '../treasury/contexts/TreasuryContext'
+import { useAccount } from 'wagmi'
+import { TopHeroComponentNotConnected, CollectiveBalancesSection } from './components'
+
+export default function Home() {
+  const { isConnected } = useAccount()
+  const { activeProposals, data } = useGetProposalsWithGraph()
+
+  return (
+    <div className="bg-bg-100 px-6">
+      {!isConnected ? (
+        <>
+          <TopHeroComponentNotConnected />
+          <TreasuryContextProviderWithPrices>
+            <CollectiveBalancesSection />
+          </TreasuryContextProviderWithPrices>
+        </>
+      ) : null}
+      <LatestCollectiveSection
+        latestProposals={data.slice(0, 3)}
+        activeProposal={isConnected ? activeProposals[0] : undefined}
+      />
+      {!isConnected ? (
+        <HeroComponent
```

Comment:
For this you can either use CommunitiesSection (as it handles the not connected state already)

OR the HeroCommunitiesComponent shouldShowLearnMore />